### DISCUSSION TITLE
Autocomplete a function doesn't works well

### DISCUSSION THREAD

**Discussion by crlcrl1** [neoclide/coc.nvim#4787] [Q&A] [answered]
When I press \<CR\> to autocomplete a function, such as `printf` in C, it will select all code before the function and use \<C-j\> or \<C-k\> to chose the next or previous parameter doesn't work.
I'm using coc-clangd in vim, what can I do to make autocompletion works well?
<img width="858" alt="屏幕截图 2023-10-21 095240" src="https://github.com/neoclide/coc.nvim/assets/128894764/215c1152-8b3d-46c7-8d0b-3dab7fef52c3">

**Comment by fannheyward** [comment]
`:CocInfo` please.

**Reply by crlcrl1** [reply]
I have run `:CocInfo` and it showed this:
<img width="602" alt="屏幕截图 2023-10-23 160426" src="https://github.com/neoclide/coc.nvim/assets/128894764/18998e00-2397-4fa8-b506-c658c5df6b4c">
Looking for your early reply

**Reply by fannheyward** [reply, marked as answer]
https://github.com/neoclide/coc.nvim/issues/4599

Upgrade your vim please.

**Reply by crlcrl1** [reply]
Thank you for giving me an answer
